app.rb

```diff
@@ -101,28 +107,21 @@ def create_book
   def create_rental
     puts 'Select wich book you want to rent by its ID'
     @books.each_with_index { |book, index| puts "#{index}. Title: #{book.title}, Author: #{book.author}" }
-
     book_id = gets.chomp.to_i
-
     puts 'Select wich person will rent the book by its ID'
     @persons.each_with_index { |person, index| puts "#{index}. Name: #{person.name}, Age: #{person.age}" }
```

Comment:
### [ Optional Changes ]

It would be nice if you could consider adjusting these changes.

- [ ] When creating a rental for a book and a person, the user is not entering ID for a book or a person. You could say "Select a book you want to rent by number (not ID)" and similar for a person. 👍🏽

<img width="418" alt="Screenshot 2023-05-23 at 8 45 04 PM" src="https://github.com/jaimevillegas/oop-school-library/assets/50941074/b5b88a8a-87e5-439a-8c6c-73a469c2d5ee">